### ISSUE TITLE
Trying to create a configured room on Ejabberd generates errors

### ISSUE THREAD

**Issue by olabiniV2** [coyim/coyim#780]
If you try to create a new configured room on Ejabberd, the debug output will sometimes contain errors. For me, this looks like this. (I've replaced the account information with asterisks.)

```
ERRO[0054] Got a presence error                          account=******@*****.*** component=session error="&{{jabber:client error}  409 cancel That nickname is registered by another person {{ } } <nil> <nil> <nil> <nil> <nil> <nil> 0xc00000ea00 <nil> <nil> <nil> <nil>}" from=dfghfghdfghdfghdfghaa@conference.*****.****
```

This error doesn't generate any notification in the dialog, but afterwards, the server doesn't seem to respond. So the functionality of creating a configured room on ejabberd seems broken.

**Comment by IvanJijon** [comment]
This issue could be closed, this has been treated in issue #774.

**Comment by olabiniV2** [comment]
OK
